### PR TITLE
refactor(Blog, Pressroom Detail Page): Blog and pressroom detail page design updated

### PR DESCRIPTION
**Pull request by sharad-neptunemutual**
### Changes
- Fix blog container size
- Fix pressroom container size
- Created content wrapper for blog and pressroom detail page
- Detail page title and date alignment fix
- fix the image container size

### CODE REVIEW

File: src/components/BlogAndPressroomViewer.astro [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+import SocialShare from "../elements/SocialShare.astro";
+import Tags from "../elements/Tags.astro";
+
+import { toTagsWithHref } from "../../util/tag";
+import { BlogOrPressroom } from "../../types/enum";
+
+interface Props {
+    type: BlogOrPressroom;
```
Comment (binodnpm) [inline src/components/BlogAndPressroomViewer.astro:9]: Incorrect formatting. Use 2 spaces. Configure your editor according to `.vscode/extensions.json`

File: src/pages/pressroom/[slug].astro [hunk 2]
```diff
@@ -10,49 +10,42 @@ import { getRelatedPosts } from "../../../util/getRelated";
 import { Api } from "../../../types/enum";
```
Comment (binodnpm) [inline src/pages/pressroom/[slug].astro:11]: What did really change in this file?

File: src/components/BlogAndPressroomViewer.scss [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+
+      figcaption a:hover {
+        color: colors.$rose-400;
+      }
+    }
+  }
+}
+
+.content.wrapper .article {
+  .kg-bookmark-container {
```
[the remark this answers is not in the document]
Comment (sharad-neptunemutual) [reply src/components/BlogAndPressroomViewer.scss]: removed

File: src/pages/blog/[slug].astro [hunk 4]
```diff
@@ -10,48 +10,41 @@ import * as dns from "../../../util/dns";
 import { Api } from "../../../types/enum";
 
 interface Props {
-  post: Article;
-  related: Article[];
+    post: Article;
```
[the remark this answers is not in the document]
Comment (sharad-neptunemutual) [reply src/pages/blog/[slug].astro]: formatting fixed

File: src/pages/blog/slug.scss [hunk 5]
```diff
@@ -30,6 +30,8 @@
 @import "../../views/blog-post/BlogPost.scss";
 @import "../../views/blog-post/BlogPostHero.scss";
 @import "../../views/blog-post/RelatedPosts.scss";
+@import "../../components/BlogAndPressroomViewer.scss";
```
[the remark this answers is not in the document]
Comment (sharad-neptunemutual) [reply src/pages/blog/slug.scss:33]: Unnecessary css has been removed and kept specific styles needed for the blog and press room layout.

File: src/pages/pressroom/[slug].astro [hunk 6]
```diff
@@ -10,49 +10,42 @@ import { getRelatedPosts } from "../../../util/getRelated";
...
```
[the remark this answers is not in the document]
Comment (sharad-neptunemutual) [reply src/pages/pressroom/[slug].astro]: fixed

File: src/pages/pressroom/[slug].astro (hunk 2 above)
Comment (sharad-neptunemutual) [reply src/pages/pressroom/[slug].astro]: Spacing fixed.

File: src/scripts/blog/slug.js [hunk 7]
```diff
@@ -7,3 +7,4 @@ import '../global/index'
 // Global End
 import '../components/content-viewer/index'
```
Comment (binodnpm) [inline src/scripts/blog/slug.js:8]: Is content-viewer still required after this change?

File: src/scripts/pressroom/slug.js [hunk 8]
```diff
@@ -7,3 +7,4 @@ import '../global/index'
 // Global End
 import '../components/subscription/index'
 import '../components/content-viewer/index'
```
Comment (binodnpm) [inline src/scripts/pressroom/slug.js:9]: Is content-viewer still needed?

File: src/scripts/blog/slug.js (hunk 7 above)
Comment (sharad-neptunemutual) [reply src/scripts/blog/slug.js:8]: content-viewer file has script for `.social.share.container`, `.copy.link` which is also used in blog and pressroom detail page.

File: src/scripts/pressroom/slug.js (hunk 8 above)
Comment (sharad-neptunemutual) [reply src/scripts/pressroom/slug.js:9]: Yes, the content-viewer file has script for .social.share.container, .copy.link which is also used in blog and pressroom detail page.

**Pull request merged**